pontoon/base/management/commands/update_auth_providers.py

```diff
@@ -0,0 +1,94 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file has been copied from http://github.com/mozilla/testpilot
+# and has been modified in order to use Fxa provider from the allauth package.
+from __future__ import absolute_import
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.contrib.sites.models import Site
+from allauth.socialaccount.models import SocialApp
+from allauth.socialaccount.providers.fxa.provider import FirefoxAccountsProvider
+from allauth.socialaccount.providers.github.provider import GithubProvider
+
+
+FXA_PROVIDER_ID = FirefoxAccountsProvider.id
+GITHUB_PROVIDER_ID = GithubProvider.id
+
+
+class Command(BaseCommand):
+    help = ('Ensures an allauth application exists and has credentials that match settings')
+
+    if FXA_PROVIDER_ID:
```

Comment:
IIUC this will always be `True`?

There a lot of code repetition here, so I suggest we refactor a little:
```python
def handle():
     if `FXA`:
        update_provider(fxa_data)

     if `GITHUB`:
        update_provider(github_data)

     # Ensure the provider applies to the current default site.

def update_provider(data):
    try:
    except:
```